app/utils/window-utils.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+let { BrowserWindow } = require("electron");
+
+/**
+ * Collection of Window Utilities
+ *
+ * @class WindowUtils
+ */
+class WindowUtils {
+	/**
+	 * Attempt to open the devtools with
+	 * a fallback if the regular method fails.
+	 *
+	 * @static
+	 * @param {Electron.BrowserWindow} window
+	 * @param {Electron.OpenDevToolsOptions["mode"]} mode
+	 * @memberof WindowUtils
+	 */
+	static openDevtools(window, mode = "right") {
```

Comment:
I dont know why the mode option needs an opinionated default value, we can have null as default value so that the `openDevTools()` will open in the last used mode